src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,19 @@
-export const CommentInfo = () => <>Put the comment here</>;
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment }) => (
+  <>
```

Comment:
looks redundant

https://react.dev/reference/react/Fragment#usage

Reply:
isn't fixed. No needs to wrap `<div className="CommentInfo">` with `<>`